packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -468,6 +468,49 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
           });
           return state.bookDataKit.recorder.write(bookData);
         },
+        observeQuoteNotifier() {
+          const { state, facets } = this;
+          const { collateralBrand, bidBrand, priceAuthority } = state;
+          assertAllDefined({ collateralBrand, bidBrand, priceAuthority });
+
+          trace('observing');
+
+          void E.when(
+            E(collateralBrand).getDisplayInfo(),
+            ({ decimalPlaces = DEFAULT_DECIMALS }) => {
+              // TODO(#6946) use this to keep a current price that can be published in state.
+              const quoteNotifier = E(priceAuthority).makeQuoteNotifier(
+                AmountMath.make(collateralBrand, 10n ** BigInt(decimalPlaces)),
+                bidBrand,
+              );
+              void observeNotifier(quoteNotifier, {
+                updateState: quote => {
+                  trace(
+                    `BOOK notifier ${priceFrom(quote).numerator.value}/${
+                      priceFrom(quote).denominator.value
+                    }`,
+                  );
+                  state.updatingOracleQuote = priceFrom(quote);
+                },
+                fail: _reason => {
+                  trace('fail', _reason);
```

Comment:
no strong opinion here, but the value is used.
```suggestion
                fail: reason => {
                  trace('fail', reason);
```

Reply:
I improved the Trace message, and renamed it to `reason`

Reply:
same issue with `_done`